README.md

```diff
@@ -133,10 +133,10 @@ The golden snapshots in source control must be updated by the [visual-diff GitHu
 To run the tests locally to help troubleshoot or develop new tests, first install these dependencies:
 
 ```shell
-npm install @brightspace-ui/visual-diff@X mocha@Y puppeteer@Z  --no-save
+npm install @brightspace-ui/visual-diff@X  --no-save
```

Comment:
These instructions are simpler now that we're using peer dependencies. [update to our npm init library](https://github.com/BrightspaceUI/create/pull/66).